airbyte-protocol/models/src/main/java/io/airbyte/protocol/models/JsonSchemaPrimitive.java

```diff
@@ -4,11 +4,26 @@
 
 package io.airbyte.protocol.models;
 
+import com.google.common.collect.ImmutableMap;
+
 public enum JsonSchemaPrimitive {
-  STRING,
-  NUMBER,
-  OBJECT,
-  ARRAY,
-  BOOLEAN,
-  NULL;
+
+  STRING_BINARY(ImmutableMap.of("type", "string", "contentEncoding", "base64")),
```

Comment:
@tuliren can we use JsonSchemaPrimitive as is in scope of this PR or better wait until https://github.com/airbytehq/airbyte/issues/7944 will be implemented?

Reply:
Yes, I think the current implementation looks good.